### PR TITLE
Remove USECASE_AUDIO_RECORD_LOW_LATENCY since it causes libhybris to …

### PR DESCRIPTION
**Pull request by mariogrip**
…crash, It seems to not find the pcm device. This is an issue on both oneplus and fairphone.

### CODE REVIEW

File: hal/audio_hw.c [hunk 1]
```diff
@@ -18,7 +18,7 @@
  */
 
 #define LOG_TAG "audio_hw_primary"
-//#define LOG_NDEBUG 0
+#define LOG_NDEBUG 0
```
Comment (peat-psuwit) [inline hal/audio_hw.c:21]: You left the debug flag enabled. This should not be in in the commit.

**Review by peat-psuwit — COMMENTED** [review]
I think the debug flag should be left enabled in the repo. Otherwise looking good.

**Review by ghost — COMMENTED** [review]
looks good to me

File: hal/audio_hw.c (hunk 1 above)
Comment (Flohack74) [reply hal/audio_hw.c:21]: @mariogrip can you re-comment this line or should I ? can we merge then?

**Comment by Flohack74** [comment]
Reminder, this is pending for one year now nearly...

**Comment by mariogrip** [comment]
No need anymore

**Pull request closed without merging**